### PR TITLE
Add formula for `doas.rb`

### PR DESCRIPTION
**Pull request by Neved4**

### CODE REVIEW

**Comment by 0risc** [comment]
The test I wrote doesn't work and I'm not sure why, but otherwise this should at least build and install `doas`.

**Review by Neved4 — COMMENTED** [review]
Thanks kindly!

File: Formula/doas.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    url :stable
+  end
+
+  depends_on :macos
+
+  def install
+    if OS.mac?
+      # ln_s "/opt/homebrew/bin/byacc", "/opt/homebrew/bin/yacc"
+      # ENV.prepend_path "PATH", "/opt/homebrew/bin"
+    end
```
Comment (0risc) [inline Formula/doas.rb:20]:
```suggestion
```
Seems like these lines are not needed? If that's true, let's remove.

**Review by 0risc — APPROVED** [review]
Looks good!

**Pull request closed without merging**